sdk/keyvault/keyvault-keys/review/keyvault-keys.api.md

```diff
@@ -105,6 +105,7 @@ export class CryptographyClient {
 
 // @public
 export interface CryptographyClientOptions extends KeyClientOptions {
+    remoteOnly?: boolean;
```

Comment:
I'm wondering if this should be an enum. We now have three configurations, and while which constructor you use matters, having some combination that doesn't make sense (like passing a JWK today with remoteOnly = true) seems like a higher risk.

Reply:
I really like the enum idea. Some variation of `mode: Local | Remote | Hybrid` (or some other language that makes sense) with Hybrid being the default?